src/DetailsView/actions/details-view-action-message-creator.ts

```diff
@@ -365,13 +367,22 @@ export class DetailsViewActionMessageCreator extends DevToolActionMessageCreator
         this.dispatcher.sendTelemetry(TelemetryEvents.DETAILS_VIEW_OPEN, telemetryData);
     }
 
-    public editFailureInstance = (description: string, test: VisualizationType, requirement: string, id: string): void => {
+    public editFailureInstance = (
+        description: string,
```

Comment:
Does it instead seem better to pass an object which models what the failure instance contains rather than individual properties?

Reply:
done! little tricky figuring out logic how it affected failure instance panel control under different circumstances but I think it works

Reply:
Simplified a bit more--longer description on below comment. Thanks!